src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -176,15 +176,16 @@ define(function (require) {
         return this.intervalName && _.find(intervals, { name: this.intervalName });
       };
 
-      self.toIndexList = function (start, stop) {
+      self.toIndexList = function (start, stop, sortDirection) {
```

Comment:
Can you add tests for this directional behavior to the existing `toIndexList` test? They'll help catch issues like the `sortDirection.reverse()` thing I pointed out below.